test/definitions/divorce/CaseTypeTab.test.js

```diff
@@ -0,0 +1,112 @@
+const expect = require('chai').expect;
+const assert = require('chai').assert;
+const _ = require('loadsh');
+const caseTypeTab = require('../../../definitions/divorce/json/CaseTypeTab');
+const caseField = require('../../../definitions/divorce/json/CaseField');
+describe('CaseTypeTab', function() {
+  describe('duplicates', function() {
+    it('should contain a unique case field ID per tab ID (no duplicate field in a tab)', function() {
+      const uniqResult = _.uniqWith(
+        caseTypeTab,
+        (field1, field2) =>
+          field1.TabID === field2.TabID &&
+          field1.CaseFieldID === field2.CaseFieldID
+      );
+      expect(uniqResult).to.eql(caseTypeTab);
+    });
+    it('should contain a unique tab field display order ID field tab ID (no duplicate field order in a tab)', function() {
+      const tabIds = _.uniq(_.map(caseTypeTab, 'TabID'));
+      tabIds.forEach(tabId => {
+        const allFieldsPerTab = _.filter(caseTypeTab, (field) => {
+          return field.TabID === tabId;
+        });
+        const uniqResults = _.uniqWith(
+          allFieldsPerTab,
+          (field1, field2) =>
+            field1.TabFieldDisplayOrder === field2.TabFieldDisplayOrder
+        );
+        expect(uniqResults).to.eql(allFieldsPerTab);
+      });
+    });
+  });
+
+  describe('sequence', function() {
+    it('should contain a proper sequence for TabFieldDisplayOrder with no gaps', function() {
+      const tabIds = _.uniq(_.map(caseTypeTab, 'TabID'));
```

Comment:
We could make this a bit tidier by importing only the lodash functions we need:
`import { uniq, map } from 'lodash';`
```
const tabIds = uniq(map(caseTypeTab, 'TabID'));
```

Reply:
Done 👍